src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -513,49 +532,55 @@ def init_namespace_sync_d_interface_tables(dbs):
         if_id_map = {}
         oid_sai_map = {}
         oid_name_map = {}
+        if_oid_namespace = {}
 
         """
         all_ns_db - will have db_conn to all namespace DBs and
         global db. First db in the list is global db.
         Ignore first global db to get interface tables if there
         are multiple namespaces.
         """
-        for db_conn in Namespace.get_non_host_dbs(dbs):
+        for inst in range(Namespace.get_start_idx_of_non_host(dbs), len(dbs)):
```

Comment:
>inst [](start = 12, length = 4)

ns_index or ns ?

Reply:
ns_index; basically, the index in db array.
index is not same as namespace, index 0 will be host namespace.

Reply:
Then call it `db_index` ?

---
In reply to: [449314977](https://github.com/Azure/sonic-snmpagent/pull/137#discussion_r449314977) [](ancestors = 449314977)

Reply:
updated as per comment